packages/dioxus-blitz/src/window.rs

```diff
@@ -268,10 +333,23 @@ impl<'a, Doc: DocumentLike> View<'a, Doc> {
         }
     }
 
+    #[cfg(feature = "accesskit")]
+    pub fn handle_accessibility_event(&mut self, event: &accesskit_winit::WindowEvent) {
+        match event {
+            accesskit_winit::WindowEvent::InitialTreeRequested => self.build_accessibility_tree(),
+            accesskit_winit::WindowEvent::AccessibilityDeactivated => {
+                // TODO
+            }
+            accesskit_winit::WindowEvent::ActionRequested(_req) => {
+                // TODO
+            }
```

Comment:
Might it be worth at least implementing "click" here (which of course is the only event we support generally. I think the code could just be copied from the regular click handler (skipping the bit which determines which node to generate the event for as we should be given a node id directly)